tests/ui/const-generics/mgca/explicit_anon_consts-7.rs

```diff
@@ -0,0 +1,10 @@
+#![feature(min_generic_const_args)]
```

Comment:
all the `explicit_anon_consts-1..6` tests should be able to be moved back into `explicit_anon_const.rs` now. ie delete them all and uncomment all the commented out tests in `explicit_anon_consts.rs`.